static/components/AppAround.js

```diff
@@ -108,8 +108,12 @@
 	        });
 	      });
 	      navigator.geolocation.getCurrentPosition(function (position) {
+	        var userPosition = [position.coords.longitude, position.coords.latitude];
+	        moveTo(userPosition);
+	        currentMarker(userPosition);
+	        console.log(userPosition);
```

Comment:
remove logs if you don't need to log (I suppose you were debugging with that, in which case it's more explicit to use `console.debug()`)